rotkehlchen/icons.py

```diff
@@ -91,6 +91,9 @@ def __init__(self, data_dir: Path, coingecko: Coingecko) -> None:
         self.custom_icons_dir.mkdir(parents=True, exist_ok=True)
         self.failed_asset_ids: set[str] = set()
 
+        avatars_dir = self.icons_dir / 'avatars'
```

Comment:
Why is this needed? Existence of avatars directory is already ensured (at the moment of querying an avatar)